packages/document/builder-doc/docs/en/config/dev/live-reload.md

```diff
@@ -0,0 +1,19 @@
+- **Type:** `boolean`
```

Comment:
Please add the document to main-doc instead of builder-doc.

builder-doc is no longer maintained

Reply:
@2heal1 cc